examples/player/simid_player.js

```diff
@@ -440,11 +467,14 @@ class SimidPlayer {
   
   /** The creative wants to play video. */
   onRequestPlay(incomingMessage) {
-    this.adVideoElement_.play().then(
-      // The play function returns a promise.
-      this.simidProtocol.resolve(incomingMessage),
-      this.simidProtocol.reject(incomingMessage)
-    );
+    
+    if (this.isLinearAd_) {
+      this.adVideoElement_.play()
+      .then(() => this.simidProtocol.resolve(incomingMessage))
+      .catch(() => this.simidProtocol.reject(incomingMessage));
```

Comment:
now is probably a good time to add a reason for the reject message.

maybe use 1206 - The SIMID media could not be loaded.